enclave/src/test/on_chain_ocall_tests.rs

```diff
@@ -0,0 +1,49 @@
+/*
+	Copyright 2019 Supercomputing Systems AG
+	Copyright (C) 2017-2019 Baidu, Inc. All Rights Reserved.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+
+use crate::ocall::{
+	ocall_api::{EnclaveOnChainOCallApi, WorkerRequest, WorkerResponse},
+	ocall_component_factory::{OCallComponentFactory, OCallComponentFactoryTrait},
+};
+use log::*;
+use std::vec::Vec;
+use substrate_api_client::utils::storage_key;
+
+#[allow(unused)]
+fn test_ocall_worker_request() {
```

Comment:
Now the we have mocks, this test should use an 'on_chain_ocall_mock' such that we don't need the node.

Reply:
there are probably quite a few tests that can make use of mocks now - but so far I did not dare to change any of those tests. This test in particular I think only makes sense if we send a real worker request? Otherwise it's completely obsolete, because it doesn't test anything else?

Reply:
Haha,  you are probably right. :)

Reply:
It's essentially an integration test - however, because of that nature, it's not active/run at the moment